### ISSUE TITLE
[android] Accessibility issues (including Talkback)

### ISSUE THREAD

**Issue by roptat** [organicmaps/organicmaps#1930]
Hi!

Today I presented the app to a friend who's blind. It was a bit hard, but with my guidance, he was able to use the navigation. However, in the process we found a few issues.

First, the menu button (bottom right) is vocalized "toggle" which makes it hard to understand what it does. Then, after selecting a point, the buttons to create navigation (from, to, favorite, ...) are vocalized almost correctly, but they are preceded with "return" or maybe "back" (that's "retour" in the French translation).

Finally, after starting the navigation, it is not possible to select a profile (car, foot, bike, public transport) since they are all vocalized as "checkbox", "checkbox", "checkbox" and "checkbox" respectively (:

Hope this helps!

**Comment by biodranik** [comment]
Was it iOS or Android? Was it a French locale?

**Comment by roptat** [comment]
It was on android, with French locale

**Comment by biodranik** [comment]
How can we reproduce these issues and hear what was pronounced?

**Comment by roptat** [comment]
You'll need an accessibility app like talkback to be able to hear what was pronounced. You might already have it or an equivalent service, check in the accessibility options. You'll also need to make sure TTS is properly set up.

**Comment by biodranik** [comment]
Looks like specific guidelines for Talkback should be followed for all on-screen elements.

**Comment by biodranik** [comment]
> Currently, when Talkback is enabled, the individual
controls can only be accessed by swiping right or left with a finger.

> This means that the map point cannot be clicked on.
So something needs to be changed in the app.
Kind regards
